### DISCUSSION TITLE
How zed capturing `CMD+Enter` keyboard-event.

### DISCUSSION THREAD

**Discussion by kingwingfly** [zed-industries/zed#12541] [Feature Requests]
I notice that zed uses `CMD+Enter` to send message to chatGPT in assistant. 

I happend to working on a CLI tool, but all crates about keyboard-event I tried cannot capture `CMD+Enter` event. And it's quite hard for me to find the part in `zed` repo currently. 

It's so kind if anyone can point out the maze.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
